### DISCUSSION TITLE
Allow copying mods in the Daily Challenge

### DISCUSSION THREAD

**Discussion by FibreTTP** [ppy/osu#29107] [Ideas] [answered]
In the same way you can right-click to copy mods in the song select leaderboard:
![2024-07-26_22 07_osu!_pJYffR9Q](https://github.com/user-attachments/assets/eb0f0ed9-2ab3-490d-9cca-c8f6752a7362)

you should also be able to right-click to "use these mods" in the Daily Challenge screen leaderboard and Daily Challenge results screen leaderboard (and also the normal osu! results screen leaderboard):

(here)
![2024-07-26_22 12_osu!_bXV6UEvj](https://github.com/user-attachments/assets/b0df927b-de4b-43ce-8e27-d6dc4ae98d6e)

(and here)
![2024-07-26_22 13_osu!_8m8wyUIB](https://github.com/user-attachments/assets/46955514-8e11-4fbe-ace0-9300c4ca23dd)

**Comment by Joehuu** [comment, marked as answer]
Working on this, but maybe I won't implement it in results screen (there's no place to put current mods).

**Reply by FibreTTP** [reply]
Cool

**Reply by Joehuu** [reply]
You can see my changes / look at progress here: https://github.com/ppy/osu/pull/29163
